TeleSharp.Generator/Program.cs

```diff
@@ -298,15 +499,16 @@ public static string GetTypeName(string type)
                     return "TLObject";
                 case "x":
                     return "TLObject";
+                case "vector t":
+                    return "List<T>";
             }
 
-            if (type.StartsWith("Vector"))
+            if (type.StartsWith("Vector<"))
```

Comment:
The previous code didn't work with "vector t" type, I made it like this to make it work with "vector t" type.